css/style.css

```
@@ -0,0 +1,505 @@
+/* Fonts */
+
+@font-face {
+  font-family: "Cuprum";
+  src: url(../fonts/cuprum.woff) format("woff"),
+    url(../fonts/cuprum.woff2) format("woff2");
+  font-weight: 400;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Cuprum";
+  src: url(../fonts/cuprumbold.woff) format("woff"),
+    url(../fonts/cuprumbold.woff2) format("woff2");
+  font-weight: 700;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Cuprum";
+  src: url(../fonts/cuprumbolditalic.woff) format("woff"),
+    url(../fonts/cuprumbolditalic.woff2) format("woff2");
+  font-weight: 700;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: "PT Sans";
+  src: url(../fonts/ptsans.woff) format("woff"),
+    url(../fonts/ptsans.woff2) format("woff2");
+  font-weight: 400;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "PT Sans";
+  src: url(../fonts/ptsansbold.woff) format("woff"),
+    url(../fonts/ptsansbold.woff2) format("woff2");
+  font-weight: 700;
+  font-style: normal;
+}
+
+/* Variables */
+
+:root {
+  /* Index */
+
+  --basic-color:#fff;
+  --basic-color-hover: rgba(197,197,197,0.3);
+  --basic-color-active: rgba(169,169,169,0.3);
+  --accent-color: #32425C;
+  --accent-color-dark: #293449;
+  --accent-color-active: #161D29;
+  --button-color: #EE3643;
+  --button-color-hover: #CA2C37;
+  --button-color-active: #BA2732;
+  --main-title-bg:#F9F5F0;
+  --content-bg: #F1F5F7;
+  --color-details:  #63A63E;
+  --color-details-hover: #5FBB2D;
+  --color-details-active: #518534;
+  --bg-img: #A9A9A9;
+  --footer-link-color: #FFDA96;
+  --text-color: #000000;
+  --text-color-active: #C5C5C5;
+  --link-color-active: rgba(255, 255, 255, 0.5);
+
+  --color-materials:  #FFBF47;
+  --color-instruments: #3BBCE3;
+  --color-technics: #DC91D8;
+  --color-discount: #8ED78F;
+  --color-delivery: #FFC047;
+  --promo-link-color:  rgba(255, 255, 255, 0.3);
+
+  /* Catalog*/
+
+}
+
+/* Global */
+
+body { margin: 0 }
+img { max-width: 100% }
+a { text-decoration: none; color: var(--basic-color); }
```

Тоже вопрос единообразия оформления кода. Это базовый критерий и это важно для успешной защиты проекта.
```suggestion
body {
  margin: 0;
}

img {
  max-width: 100%;
}

a {
  text-decoration: none; color: var(--basic-color);
}
```